conda-forge.yml

```diff
@@ -1,5 +1,8 @@
+build_platform:
```

Comment:
FWIW, this specifies that we're cross-compiling aarch from x64. It's possible that it's easier to start from a "native" build (still actually running on x64, but emulated through qemu - this is much slower though)